tasks/37_min-number-to-prime/min-number-to-prime.js

```diff
@@ -0,0 +1,17 @@
+export function minNumberToPrime(arr) {
+  const sum = arr.reduce((acc, cur) => acc + cur);
+  let closestPrime = sum;
+  for (let i = sum; i < 2 * sum; i++) {
+    let isPrime = true;
+    for (let j = 2; j < Math.sqrt(sum); j ++) {
```

Comment:
As I said don't hesitate to introduce other functions that may increase code readability. Here you could introduce another function `isPrimeNumber` where you will pass number and it will return Boolean.